caliban_toolbox/utils/__init__.py

```diff
@@ -30,8 +30,6 @@
 from __future__ import print_function
 
 
-from caliban_toolbox.utils import io_utils
-from caliban_toolbox.utils import misc_utils
 from caliban_toolbox.utils import utils
```

Comment:
whats the justification for combining these util files as its likely that utils will grow over time?

Reply:
Basically all of those util functions weren't used anymore. I combined all the old ones together because I think more of them are going to be culled. I think we'll be totally reorganizing the structure of the repo once we have it paired down the code that is still in use. e.g., right now npz_preprocessing and npz_postprocessing contain the bulk of them, but they aren't split up at all